### DISCUSSION TITLE
如何修改成果清单中的作者为“第一发明人”等（如奖项、专利、书籍、项目）

### DISCUSSION THREAD

**Discussion by znsoftm** [BITNP/BIThesis#547] [Q&A] [answered]
在盲审模式中，默认模板只会标注为“第一作者”，但是实际上专利叫“第一发明人”，奖项叫“第一完成人”，翻译的书籍叫“第一译者” 怎样才能在盲审模式正确显示？

**Comment by fky2015** [comment]
可以自定义 \Author。详见我们的使用手册，或者 pubs.bib 的注释。

https://github.com/BITNP/BIThesis/blob/main/templates/graduate-thesis/reference/pub.bib#L33

https://github.com/BITNP/BIThesis/blob/424e5b851df44d42cbf26900874480b5de58484a/templates/graduate-thesis/reference/pub.bib#L32-L33

![图片](https://github.com/user-attachments/assets/c81bc83e-dd91-45de-b575-2f7f0735e29c)

**Comment by znsoftm** [comment, marked as answer]
在`pub.bib`中修改 

```bibtex
author+an = {1:myself="\Author[][][第一发明人]"},
```

**Comment by YDX-2147483647** [comment]
为方便大家查看，这个问题归档到了 [🥑 疑难杂症 (bithesis.bitnp.net/faq)](https://bithesis.bitnp.net/faq.html)，后续将在那边更新。如果遇到新问题，欢迎继续在 GitHub Discussions 讨论。

https://bithesis.bitnp.net/faq/bib-author.html
https://github.com/BITNP/BIThesis-wiki/blob/96df7d463a26ed01a2000ee13e2a3b6c4f5c2452/wiki/faq/bib-author.md
